Kickstarter-iOS/Tests/Views/BackerDashboardProjectsViewControllerTests.swift

```diff
@@ -0,0 +1,84 @@
+import Prelude
+import Result
+import XCTest
+@testable import Kickstarter_Framework
+@testable import KsApi
+@testable import Library
+
+// swiftlint:disable:next type_name
+internal final class BackerDashboardProjectsViewControllerTests: TestCase {
+  fileprivate var project1: Project!
```

Comment:
possible to do this without the IUO vars? could we just put the definitions of these projects right in line?